packages/flutter/test/material/bottom_app_bar_test.dart

```diff
@@ -86,6 +86,37 @@ void main() {
     expect(physicalShape.color, const Color(0xff0000ff));
   });
 
+  testWidgets('toggle hasNotch', (WidgetTester tester) async {
+    await tester.pumpWidget(
```

Comment:
A comment explaining why we're toggling hasNotch...

Reply:
done